notebooks/ullyses-walkthrough.ipynb

```diff
@@ -617,6 +619,137 @@
     "tss_hdu.close()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Sub-exposure level TTS Data Product Example\n",
+    "\n",
+    "Next let's check out a TTS file for one of the T Tauri monitoring stars, TW-Hydra."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use curl to download a TSS data product for the target\n",
+    "!curl -L -X GET \"https://mast.stsci.edu/api/v0.1/Download/file?uri=mast%3AHLSP%2Fullyses%2Fv-tw-hya%2Fdr7%2Fhlsp_ullyses_hst_cos_v-tw-hya_g160m_dr7_split-tss.fits\" --output \"hlsp_ullyses_hst_cos_v-tw-hya_g160m_dr7_split-tss.fits\" --fail --create-dirs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a variable to point to the file we just downloaded, then open the file, and print the HDU info\n",
+    "tss_file = Path('./hlsp_ullyses_hst_cos_v-tw-hya_g160m_dr7_split-tss.fits')\n",
+    "tss_hdu = fits.open(tss_file)\n",
+    "print(tss_hdu.info())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\\\n",
+    "From the filename, we can see:\n",
+    "- the telescope used is HST\n",
+    "- the instrument used is COS\n",
+    "- the optical element is G160M\n",
+    "- this file was created for data release 7\n",
+    "\n",
+    "Now we'll look at the science and provenance extensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print the columns in the science extension\n",
+    "print(tss_hdu[1].columns)\n",
+    "\n",
+    "# Print the shape of the FLUX array\n",
+    "print('Number of time bins: {}'.format(tss_hdu[1].data['FLUX'][0].shape[0]))\n",
+    "print('Number of spectral bins: {}'.format(tss_hdu[1].data['FLUX'][0].shape[1]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print the columns in the provenance extension\n",
+    "print(tss_hdu[2].columns)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We'll use Astropy Tables to make it print out a few of the columns listed above in a neat way\n",
+    "\n",
+    "prov_table = Table(tss_hdu[2].data)\n",
+    "prov_table.pprint_include_names = ('FILENAME', 'PROPOSID', 'INSTRUMENT', 'DETECTOR', 'FILTER', 'APERTURE', 'XPOSURE', 'MJD_MID')\n",
+    "\n",
+    "# prov_table.pprint()\n",
+    "\n",
+    "# To print all columns use:\n",
+    "prov_table.pprint_all()\n",
+    "\n",
+    "# Or to print a single column use:\n",
+    "# prov_table['FILENAME'].pprint()"
```

Comment:
So I left these in for all examples but commented out so that folks could see different options for printing out the table. Maybe I can add a comment to make that more clear

Reply:
Ok! I broke them into different cells above, but I think that works because it shows what it can do the first time and then later on they can decide to use it or not if you agree.